File: app/src/main/java/com/egoriku/catsrunning/helpers/TypeFit.java
package com.egoriku.catsrunning.helpers;

import android.support.annotation.IntDef;

import java.lang.annotation.Retention;
import java.lang.annotation.RetentionPolicy;

@Retention(RetentionPolicy.SOURCE)
@IntDef({TypeFit.UNCERTAIN, TypeFit.WALKING, TypeFit.RUNNING, TypeFit.CYCLING})
public @interface TypeFit {
    int UNCERTAIN = 0;
    int WALKING = 1;
    int RUNNING = 2;
    int CYCLING = 3;
}

File: app/src/main/java/com/egoriku/catsrunning/helpers/TypeFit.kt
package com.egoriku.catsrunning.helpers

import android.support.annotation.IntDef

class TypeFit {

    companion object {
        const val WALKING = 1L
        const val REMINDER = 2L
        const val MAIN = 3L
    }

    @Retention(AnnotationRetention.SOURCE)
    @IntDef(WALKING, REMINDER, MAIN)
    annotation class TypeFit
}

